vocode/streaming/synthesizer/eleven_labs_synthesizer.py

```diff
@@ -89,6 +90,7 @@ async def create_speech(
 
                 audio_segment.export(output_bytes_io, format="wav")  # type: ignore
 
+                create_speech_span.end()
```

Comment:
can we split up the measurement to the api response + conversion? bc now we don't know the actual breakdown

Reply:
Done